test/src/test/java/hudson/model/labels/LabelExpressionTest.java

```diff
@@ -207,8 +210,10 @@ public void laxParsing() {
     @Test
     public void dataCompatibilityWithHostNameWithWhitespace() throws Exception {
         assumeFalse("Windows can't have paths with colons, skipping", Functions.isWindows());
-        DumbSlave slave = new DumbSlave("abc def (xyz) test", "dummy",
-                j.createTmpDir().getPath(), "1", Mode.NORMAL, "", j.createComputerLauncher(null), RetentionStrategy.NOOP, Collections.EMPTY_LIST);
+        DumbSlave slave = new DumbSlave("abc def (xyz) test", tempFolder.toString(), j.createComputerLauncher(null));
+        slave.setRetentionStrategy(RetentionStrategy.NOOP);
+        slave.setNodeDescription("dummy");
+        slave.setNodeProperties(new ArrayList<>());
```

Comment:
Prefer immutable lists to mutable lists where possible:

```suggestion
        slave.setNodeProperties(List.of());
```

Reply:
I used the `Collections.EMPTY_LIST` as before. I guess this is also fine, because it is also immutable. But thanks for the hint.